index.js

```diff
@@ -4,13 +4,61 @@ var pluralize = require("pluralize");
 module.exports = function(app, handler){
 	return {
 		addResource : function(name){
-			app.use("/" + pluralize(name), function(req, res, next){
-				handler.findAll(S("_" + name).camelize().s)
-				.then(function(records){
-					res.json(records);
-				})
-				.catch(next);
-			});
+			var resourceName = S("_" + name).camelize().s;
+
+			var buildRoutes = function(name){
+				var router = require("express").Router();
+
+				router.param("id", function(req, res, next, id){
```

Comment:
I believe that express still doesn't isolate params per router, so this won't work. Check the latest version and see if that's still true.